.github/workflows/status.yaml

```diff
@@ -0,0 +1,64 @@
+name: Custom GH Status from Workflow Artifacts
+
+on:
+  workflow_run:
+    workflows: ["pr"]
+    types:
+      - completed
+
+jobs:
+  process_artifacts:
```

Comment:
We should probably be explicit about the permissions required for this job.

Can you add a `permissions` key?

See here for details: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idpermissions

Reply:
Done 👍